patches/minecraft/net/minecraft/client/gui/GuiMultiplayer.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/client/gui/GuiMultiplayer.java
++++ b/net/minecraft/client/gui/GuiMultiplayer.java
+@@ -296,7 +296,7 @@
```

Comment:
Where's the other half of this patch? https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/patches/minecraft/net/minecraft/client/gui/GuiMultiplayer.java.patch#L7

Reply:
Somehow completely missed this, whoops